src/App/index.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Alert, Button } from '@patternfly/react-core';
 import './app.css';
+import './pf-ie11.css';
```

Comment:
When I am running a yarn build, I am getting a "module not found"  error when looking for this import. Are you seeing this too?

Reply:
Ah, silly me - I should have mentioned to first run the new target `bootstrap-ie`. This generates that pf-ie11.css file and the assets directory